naive_lidar_clustering_node:
  ros__parameters:
    use_sim_time: true
    clustering:
      x_bound: 15.0
      y_bound: 15.0
      z_bound: 5.0
      downsample_voxel_size: 0.05
      remove_ground_plane_thres: 0.5
      clustering_min_points: 10.0
      clustering_max_points: 10000.0
      clustering_max_step_size: 0.5
      sacle_up_objects: 1.0
      N: 20 # dynamic object horizon
      dt: 0.2 # dynamic object horizon time step
      min_cluster_height: 0.6
    